### PR TITLE
Google.WebDesigner version 1.3.36.122

### PR DESCRIPTION
**Pull request by deadlynjdj**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/56783)

### CODE REVIEW

File: manifests/g/Google/WebDesigner/1.3.36.122/Google.WebDesigner.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+Tags:
+- google
+- web
+- web designer
+- website maker
+- web bulider
+- free
+Agreements:
+- AgreementUrl: https://policies.google.com/terms
+- AgreementUrl: https://policies.google.com/privacy
```
Comment (ItzLevvie) [inline manifests/g/Google/WebDesigner/1.3.36.122/Google.WebDesigner.locale.en-US.yaml:29]: This can only be used by Google employees.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/g/Google/WebDesigner/1.3.36.122/Google.WebDesigner.locale.en-US.yaml (hunk 1 above)
Comment (jedieaston) [reply manifests/g/Google/WebDesigner/1.3.36.122/Google.WebDesigner.locale.en-US.yaml:29]: Are you sure? I can download it and use it fine from a public landing page (https://webdesigner.withgoogle.com)

There's a forum and everything: https://support.google.com/webdesigner/threads?hl=en&thread_filter=(category:getting_started)

File: manifests/g/Google/WebDesigner/1.3.36.122/Google.WebDesigner.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+PrivacyUrl: https://policies.google.com/privacy?hl=
+Author: Google
+PackageName: Google Web Designer
+PackageUrl: https://webdesigner.withgoogle.com/
+License: Copyright 2018 Google LLC
+LicenseUrl: https://policies.google.com/terms?hl=
+Copyright: https://policies.google.com/terms?hl=
+ShortDescription: Create engaging, interactive HTML5-based designs and motion graphics that can run on any device.
+Description: Google Web Designer gives you the power to create beautiful, engaging HTML5 content. Use animation and interactive elements to bring your creative vision to life, and enjoy seamless integration with other Google products, like Google Drive, Display & Video 360, and Google Ads.
+Moniker: Google Web Designer
```
Comment (jedieaston) [inline manifests/g/Google/WebDesigner/1.3.36.122/Google.WebDesigner.locale.en-US.yaml:19]:
```suggestion
Moniker: google-web-designer
```

**Review by jedieaston — CHANGES_REQUESTED** [review]

File: manifests/g/Google/WebDesigner/1.3.36.122/Google.WebDesigner.locale.en-US.yaml (hunk 1 above)
Comment (ItzLevvie) [reply manifests/g/Google/WebDesigner/1.3.36.122/Google.WebDesigner.locale.en-US.yaml:29]: The `Agreements` field isn't meant contributors; only publishers.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
See https://github.com/microsoft/winget-pkgs/pull/56783#discussion_r842592967.

File: manifests/g/Google/WebDesigner/1.3.36.122/Google.WebDesigner.locale.en-US.yaml (hunk 1 above)
Comment (jedieaston) [reply manifests/g/Google/WebDesigner/1.3.36.122/Google.WebDesigner.locale.en-US.yaml:29]: Oh, I thought you meant the app in general, not the field. Sorry!

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
image

**Pull request closed without merging**